server/config/config.go

```diff
@@ -1658,3 +1658,13 @@ func SetTestMDMConfig(t testing.TB, cfg *FleetConfig, cert, key []byte, appleBMT
 	cfg.MDM.AppleSCEPSignerValidityDays = 365
 	cfg.MDM.AppleSCEPChallenge = "testchallenge"
 }
+
+// Undocumented feature flag for Windows MDM, used to determine if the Windows
+// MDM feature is visible in the UI and can be enabled. More details here:
+// https://github.com/fleetdm/fleet/issues/12257
+//
+// TODO: remove this flag once the Windows MDM feature is ready for
+// release.
+func IsMDMFeatureFlagEnabled() bool {
```

Comment:
The rationale to have it as a function here (and not just in the response payload of `GET /config`) is that we may need to check it in the backend too if we support enabling Windows MDM via `fleetctl apply`.